release-notes/opendistro-for-elasticsearch-security.release-notes-1.13.0.0.md

```diff
@@ -34,3 +36,9 @@ Compatible with Elasticsearch 7.10.2
 * Refactor Resolved ([#929](https://github.com/opendistro-for-elasticsearch/security/pull/929))
 * Combine log messages of no cluster-level permission ([#1002](https://github.com/opendistro-for-elasticsearch/security/pull/1002))
 * Support ES 7.10.2 ([#1005](https://github.com/opendistro-for-elasticsearch/security/pull/1005))
+* Add the Linux Foundation's Developer Certificate of Origin in pull request template ([#1022](https://github.com/opendistro-for-elasticsearch/security/pull/1022))
```

Comment:
Should it be in a different order?

Reply:
Discussed with @vrozov, the order doesn't matter much.

Reply:
Updated the sequence.